WordPressKit/WordPressKit.podspec

```diff
@@ -30,4 +30,25 @@ Pod::Spec.new do |s|
   # Use a loose restriction that allows both production and beta versions, up to the next major version.
   # If you want to update which of these is used, specify it in the host app.
   s.dependency 'WordPressShared', '~> 2.0-beta'
+
+  s.test_spec 'Tests' do |test_spec|
+    test_spec.dependency 'OHHTTPStubs', '~> 9.0'
+    test_spec.dependency 'OHHTTPStubs/Swift', '~> 9.0'
+    test_spec.dependency 'OCMock', '~> 3.4'
+    test_spec.dependency 'Alamofire', '~> 5.0'
+
+    test_spec.source_files = 'Tests/**/*.{h,m,swift}'
+    test_spec.resource_bundles = {
+      # The files in this bundle are duplicated at the root of the test bundle,
+      # because some tests are still looking for them at the root directory,
+      # instead of the `CoreAPITests` bundle.
+      'CoreAPITests' => 'Tests/CoreAPITests/Stubs/**/*',
+    }
+    test_spec.resources = [
+      'Tests/WordPressKitTests/Mock Data/**/*',
+      'WordPressKitTests/**/*',
+      'Tests/CoreAPITests/Stubs/**/*',
+      'Tests/**/*.{json,html,xml}',
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/a82a815b1276904f94207c5eae5e2fc86654aeb6/WordPressKit/WordPressKit.podspec#L51"></span>Style/TrailingCommaInArrayLiteral: Avoid comma after the last item of an array.</td>
    </tr>
  </tbody>
</table>